Teacher Workout/Features/ThemeDetails/View/ThemeDetails.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+struct ThemeDetails: View {
+    @StateObject var viewModel: ThemeDetailsViewModel
```

Comment:
I think this needs to be an ObervedObject if we just pass a reference and not instantiate here.